packages/money_transfer_service/config/config.dev.js

```diff
@@ -0,0 +1,6 @@
+const { PORT } = process.env;
+
+module.exports = {
+  port: PORT || 4000,
+  mongoURI: `mongodb://localhost/trans/moneytrans`
```

Comment:
This should be the port and the database not the table name so it will be:
```suggestion
  mongoURI: `mongodb://localhost:27017/trans`
```
The port is 27017 because mongodb runs on that port.

Also change the database name to Transaction then your string will be:
```suggestion
  mongoURI: `mongodb://localhost:27017/Transaction`
```